runtime_local/src/run-vehicledatabroker-cli.sh

```diff
@@ -18,6 +18,12 @@ echo "### Running VehicleDataBroker CLI                   ###"
 echo "#######################################################"
 
 SCRIPT_PATH=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+ROOT_DIRECTORY=$( realpath "$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )/../../" )
+CONFIG=$(cat $ROOT_DIRECTORY/runtime.json | jq '.[]| select(.id == "vehicledatabroker").config')
+
+DATABROKER_IMAGE=$(echo $CONFIG | jq '.[] | select(.key == "image").value')
```

Comment:
Shall we protect somehow implementation if runtime.json is unavailable or configuration doesn't contain necessary keys?

Reply:
hmm good point, but I guess the databroker is somehow mandatory in our case so I would like to avoid any additional effort